.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+indent_style = space
+indent_size = 3
```

Comment:
why this?

Reply:
I mentioned it in the OP too; lots of files seem to have 3 indentation size, rather than even, so went with it to make sure its consistent - reindenting to 2 or 4 is something I can do as a follow-up (going to be braindead work lol)

Reply:
oh okay, let's followup on this 😄